ports/stm32/boards/Passport/modules/tasks/delete_directory_task.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2023 Foundation Devices, Inc. <hello@foundationdevices.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# delete_directory_task.py - Task to delete a directory recursively
+
+
+async def delete_directory_task(on_done, full_path):
```

Comment:
What's the use case for this?  I'm not that comfortable with Passport being able to obliterate the contents of an SD card.  Seems like an easy way to lose your backups.

Reply:
I put in a warning page for deleting any directory that isn't empty